.gitignore

```diff
@@ -16,3 +16,4 @@ test/tmp
 test/version_tmp
 tmp
 spec/fixtures/test-opsbots
+.idea/*
```

Comment:
You can create a global gitignore on your local machine for this.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Don't need to remove this line since we also declaring it in the logstash `.gitignore`.